### ISSUE TITLE
NTR: 5-hydroxymethyluracil high in CSF (Lab: cerebrospinal fluid tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8466]
New term request 5-hydroxymethyluracil high in CSF (elthat_111121043052): Cerebrospinal fluid

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000208
- label: Elevated CSF 5-hydroxymethyluracil concentration
- parents: [Abnormal CSF metabolite concentration (HP:0025454)](https://hpo.jax.org/app/browse/term/HP:0025454)
- pmid: PMID:35249459
- def: The concentration of 5-hydroxymethyluracil in the cerebrospinal fluid (CSF) is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8466
- synonyms: 5-hydroxymethyluracil high in CSF(NONE)
